packages/documentation-site/patternfly-docs/content/design-guidelines/content/writing-patternfly-design-guidelines.md

```diff
@@ -1,74 +1,74 @@
 ---
-id: Writing design guidelines for PatternFly
+id: Writing PatternFly design guidelines
 section: UX writing
 ---
 
-Design guidelines walk through building design patterns with PatternFly. This page focuses on how to write clear and consistent design documentation. To learn how to contribute design guidelines, see our [contribution guide](https://github.com/patternfly/patternfly-org/wiki/Contributing-to-patternfly-org-for-designers).
+This guide provides instructions for writing clear and consistent design documentation for PatternFly. PatternFly's design guidelines provide users with information regarding the design, usage, behavior, and more of PatternFly's elements and components.
 
-## Design guidelines template
-When creating guidelines, information must be presented in the following order. Sections may be omitted, but if present must be under one of the following headings.
+To learn how to contribute design guidelines, see our [contribution guide](https://github.com/patternfly/patternfly-org/wiki/Contributing-to-patternfly-org-for-designers).
+
+## Design guidelines sections
+When creating guidelines, information must be presented in the following order. Sections may be omitted, but if present must use the following section headings (H2).
+
+### Elements 
+Display an image of the component containing numbered annotation tags for each major part of the component's visual design. Include specific icons, headers, fields, symbols, and anything else included in the component's structure. Following the image, list all the parts you tagged, aligning with the number displayed in the respective annotation tag. In this list, provide a description of each one. 
 
-### Elements (H2)
-Introduce the component and list the primary parts that make up its structure. This includes specific icons, headers, fields, symbols, or anything else included in the component.
 Example:
 
-<img src="./img/sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="479px" />
+<img src="./img/sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="600px" />
```

Comment:
Not sure why, but this image looks so small.. try removing the "px" from the width?

Reply:
I updated it to 750 and it looks a little better. Do you think the image is too busy for what it's trying to demonstrate? I'm 50/50. Could be better to show a simpler one like [chip](https://patternfly-org-pr-3637-site.surge.sh/components/chip/design-guidelines) or I could make a more abstract image. IDK this may be splitting hairs haha maybe I just get tripped up by this image when reading the page because my brain doesn't register it as an example at first

Reply:
@edonehoo I agree that it might be a lot, and it kinda blends into the rest of the guidelines in a weird way, so it also wasn't apparent to me that it was an example at first haha so yeah maybe try switching it out with a simpler version of that example (like chips) and add a white background to the artboard maybe so it stands out more from the rest of the page?